languages/go/config.json

```
@@ -9,7 +9,26 @@
   },
   "ignore_pattern": "example(?!.*test)",
   "exercises": {
-    "concept": [],
+    "concept": [
+      {
+        "slug": "basic-strings",
```

The slug doesn't currently match the concept of the exercise (`strings`). This is not a requirement, but it _is_ common to have the two align. You could thus consider renaming the slug to:

```suggestion
        "slug": "strings",
```

Do note that this means that the `exercises/concept/basic-strings` directory also has to be renamed to `strings`.

The same goes for the `basic-slices` slug below. If you'd like to do this, I would suggest doing it in a follow-up PR as to not make this PR too big.